src/Chapter06/Listing06.53.DefiningAPartialMethodToAccessItsImplementation.cs

```diff
@@ -14,8 +13,8 @@ public partial class Person
         #region HIGHLIGHT
         static partial void OnLastNameChanging(string value);
         static partial void OnFirstNameChanging(string value);
-        #endregion
-        #endregion
+        #endregion HIGHLIGHT
+        #endregion Extensibility Method Definitions
```

Comment:
```suggestion
        #endregion
```
I don't think we want this here since it's not an include, highlight, or exclude endregion. Thoughts @Keboo?